metabase/metabase
# Use sentence case for temporal bucket names

Epic #30509

In MLv1, temporal bucket option names use title case (example: "Day of Month"). OTOH, notebook cells use sentence case (example: "Created At: Day of month"). After a double-check with the design team, Maz [said](https://metaboat.slack.com/archives/C01LQQ2UW03/p1684431608938659) we should prefer using sentence case everywhere (example "Day of month"). MLv2 already does that, and this PR makes MLv1 use sentence case as well to make things consistent

### To Verify

1. New > Question > Sample Database > Orders
2. Click the "summarize" step icon to add a new step
3. Click "Pick a column to group by"
4. Hover the "Created At" column
5. Click "Month" on the right side to open a list of temporal units
6. Ensure all units use sentence case (e.g. "Month of year" instead of "Month of Year")



- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30875)
